services/core-web/src/components/mine/Incidents/MineIncidentTable.js

```diff
@@ -168,6 +176,16 @@ export class MineIncidentTable extends Component {
           </div>
         ),
       },
+      {
+        title: "Incident Type(s)",
+        key: "incident_types",
+        dataIndex: "incident_types",
+        render: (text) => (
+          <div title="Commodity">
```

Comment:
Update div `title` to match column title.

Reply:
changed